packages/ui/src/components/Session/PreSession/PreSession.tsx

```diff
@@ -3,7 +3,8 @@ import { Ready } from "@/components/Session/Ready";
 import { IUser, Void } from "@litespace/types";
 import Actions from "@/components/Session/Actions";
 import cn from "classnames";
-import FocusedStream from "@/components/Session/FocusedStream";
+import Stream from "@/components/Session/Stream";
+import { SessionType } from "@/components/Session/types";
```

Comment:
You can access the session type from `ISession.Type`. No need to redefine it. Please remove the new one and make sure all references are updated.